dask/array/percentile.py

```diff
@@ -91,8 +91,6 @@ def percentile(a, q, interpolation="linear", method="default"):
     --------
     numpy.percentile : Numpy's equivalent Percentile function
     """
-    from dask.dataframe.dispatch import _percentile
```

Comment:
Not going through the dispatch `_percentile` and instead going through local `_percentile` will it not result in dispatch of the registered backends never being used ?

Reply:
That's correct. I was initially concerned with making whatever were needed to remove the `pandas` dependency in `dask.array`. I agree we may need further updates to get back the functionality added in https://github.com/dask/dask/pull/8029. 

Since `dask.dataframe` depends on NumPy and pandas, we can import `dask.array` code inside `dask.dataframe`, but not the other way around. Is there a way we could move that dispatching logic from `dask.dataframe` into `dask.array` instead?

Reply:
> Is there a way we could move that dispatching logic from `dask.dataframe` into `dask.array` instead?

I think we can move these to `dask.array` dispatch and backend: https://github.com/dask/dask/blob/main/dask/dataframe/dispatch.py#L25-L29
&
https://github.com/dask/dask/blob/main/dask/dataframe/backends.py#L538-L540

Reply:
Would you mind pushing up a PR which handles the `dask.dataframe` -> `dask.array` migration?

Reply:
Opened a PR: https://github.com/dask/dask/pull/8083, could you review it?

Reply:
cc: @quasiben

Reply:
Thanks @galipremsagar! Just left comments over in https://github.com/dask/dask/pull/8083